docs/spec.md

```diff
@@ -2873,7 +2874,7 @@ behavior is undefined. More formally, for all `id < jd` from `indices(result)`,
     in `slice_sizes` corresponding to `collapsed_slice_dims` are not included.
   * `combine` puts `batch_dim_sizes` at axes corresponding to `batch_dims` and
    `offset_dim_sizes` at axes corresponding to `offset_dims`.
-* (C15) `operand` and `result` have the same element type.
+* (C14) `operand` and `result` have the same element type.
```

Comment:
Do we also need to update the `Inputs` section now that this constraint has been renamed?

Reply:
Thanks for the catch.